packages/web/src/app/Layout.tsx

```diff
@@ -27,7 +27,7 @@ export function Layout() {
     <Suspense fallback="loading">
       <div className="min-h-screen flex flex-col">
         <Header language={language} onLanguageChange={selectLanguage} />
-        <div className="flex-grow relative">
+        <div className="flex-grow relative flex flex-col">
```

Comment:
I'm not sure it's a good idea to put flex on this wrapper element. That locks every view to use column flex whether it makes sense or not. What is this needed for?

Reply:
So that the loading spinner could be vertically centered. The `flex-grow` on line 149 of `TranslationView.tsx` works with this `flex-col`